docs/xdr/features/collect/ingestion_methods/https.md

```diff
@@ -90,10 +92,10 @@ For numerous events, you can use the alternative endpoint `/batch`. The events s
 ```python
 import requests
 
-headers = {"X-SEKOIAIO-INTAKE-KEY": "YOUR_INTAKE_KEY"}
```

Comment:
Why did you remove the previous example ? we could provide 1 example for each use case